src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
+  postId: number,
```

Comment:
Consider using semicolons instead of commas to separate interface properties for consistency with TypeScript style.

Reply:
got it